pkg/dockerregistry/server/auth_test.go

```diff
@@ -151,15 +151,27 @@ func TestAccessController(t *testing.T) {
 				},
 				Action: "pull",
 			}},
-			basicToken:        "b3BlbnNoaWZ0OmF3ZXNvbWU=",
+			basicToken: "b3BlbnNoaWZ0OmF3ZXNvbWU=",
+			openshiftResponses: []response{
+				{200, runtime.EncodeOrDie(kapi.Codecs.LegacyCodec(registered.GroupOrDie(kapi.GroupName).GroupVersions[0]), &userapi.User{ObjectMeta: kapi.ObjectMeta{Name: "usr1"}})},
+			},
 			expectedError:     ErrNamespaceRequired,
 			expectedChallenge: false,
+			expectedActions: []string{
+				"GET /oapi/v1/users/~ (Authorization=Bearer awesome)",
```

Comment:
I'd call this a regression. There's one useless request for malformed request producing no audit log message. Previously no master API request was necessary.